### ISSUE TITLE
No new post are shown while running runserver

### ISSUE THREAD

**Issue by GonxoMR** [airbnb/knowledge-repo#304]
Auto-reviewers: @NiharikaRay @matthewwardrop @earthmancash @danfrankj

Good morning, 

recently I have move the knowledge-repo from my local machine to the server where will be running. Along with this I have been creating new knowledge and I have been uploading it.  The problem is that even though the new content shows in the repo of my machine the origin in the server, it doesn't show up while running runserver locally. 
I am submitting, merging and committing the new knowledge but still no change is shown. 

Any indications?
Thank you in advance, 
Gonxo

**Comment by matthewwardrop** [comment]
Hi @GonxoMR ,

Currently the the Knowledge Repo server does not fetch changes made to the repository during the reindexing phase (that will change soon), and so your best bet for now is to run a `cron` job that fetches the repository and keeps it up to date. Once the repository has been updated on the server filesystem, the Knowledge Repo server should reindex the new posts and show them. Let me know if that is not the case.

Closing this out in favour of duplicate #295 .

**Comment by GonxoMR** [comment]
Hi @matthewwardrop,

I think my problem is different. I am not running on the server I am running dev locally. Also I restart the repo every time (killing the process and running runserver again). Although before to move to the server it would get the changes in the knowledge if I updated it. 

In addition when I updated one of the post, it disappeared from the knowledge-repo when running runserver, although it showed in the git repos. (I have submitted and merged the new branches with master).

We are using Bonobo as a git server in our organisation. My guess was that they way Bonobo treats the git repositories is interfering with the app hooks. What is your opinion?

Thank you for your time, 
Gonxo
